theseus/src/launcher/args.rs

```diff
@@ -304,24 +299,29 @@ fn parse_arguments<F>(
 where
     F: Fn(&str) -> crate::Result<String>,
 {
+    let temporary_replace_char = "\n";
```

Comment:
what is this used for? since it's used in multiple places as well it should be a constant and moved to the top of the file

Reply:
should still be a const